.github/workflows/tests.yml

```diff
@@ -13,15 +13,15 @@ jobs:
     strategy:
       matrix:
         os: [Ubuntu, MacOS, Windows]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
         experimental: [false]
         bootstrap-args: [""]
         include:
           - os: Ubuntu
             python-version: pypy3
             experimental: false
           - os: Ubuntu
-            python-version: "3.10.0-alpha - 3.10.0"
+            python-version: "3.10"
             experimental: true
```

Comment:
why is this marked `experimental` in the 'tests' workflow, but not the 'integration' workflow?

Reply:
Unlike the integration test, this points to master poetry git repo. Here's the block in full from `tests.yml`:

```yaml
          - os: Ubuntu
            python-version: "3.10"
            experimental: true
            bootstrap-args: "--git https://github.com/python-poetry/poetry.git"
```

So the question may be: Why doesn't tox also check against git master? I propose that could be better to tackle in a separate issue, but we can also do it here if there's an agreement we could come to.

`workflow/integration.yml`:

[Integration test action](https://github.com/python-poetry/poetry-core/blob/25f6ded/.github/workflows/integration.yml#L31-L33) ([tox.ini](https://github.com/python-poetry/poetry-core/blob/25f6dedcd142148daf3e2f4f495d30055715ddd7/tox.ini)) run [tests/integration/](https://github.com/python-poetry/poetry-core/tree/c6dd2487947a450b030f9649c931964ac4e4a83a/tests/integration).

The reason it had `--experimental` before may be because 3.10 support itself wasn't stable.

Reply:
that makes sense. might be good to raise a separate ticket for discussion about the potential inconsistency